### DISCUSSION TITLE
Add sliding zoom bar to Image Viewer

### DISCUSSION THREAD

**Discussion by tlammer** [specify/specify7#5527] [Feature Requests]
Requested by UMich Herbarium

" Turns out that when trying to use the Workbench to batch load 
images, the image viewer only has a couple of magnification levels for 
trying to do the transcription, so they are having to use an external 
tool which allows a finer control over magnification."

**Comment by maxpatiiuk** [comment]
Attachments are not supported in the WorkBench, are they?

Does this issue belong to Specify 6 instead?
